core/tauri/src/window/mod.rs

```diff
@@ -65,12 +66,32 @@ pub(crate) type WebResourceRequestHandler =
 pub(crate) type NavigationHandler = dyn Fn(&Url) -> bool + Send;
 pub(crate) type UriSchemeProtocolHandler =
   Box<dyn Fn(http::Request<Vec<u8>>, UriSchemeResponder) + Send + Sync>;
+pub(crate) type OnPageLoad<R> = dyn Fn(Window<R>, PageLoadPayload<'_>) + Send + Sync + 'static;
 
 #[derive(Clone, Serialize)]
 struct WindowCreatedEvent {
   label: String,
 }
 
+/// The payload for the [`OnPageLoad`] hook.
```

Comment:
the `OnPageLoad` type is not exported and so either link to a public method or Just make this type public